### PR TITLE
Testing is fun!

### PR DESCRIPTION
**Pull request by kreeve**

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -97,7 +97,8 @@
```
Comment (MiloDavis) [inline build.xml:97]: Add exclude mokito from jar

File: src/com/nutrons/framework/controllers/CanControlMode.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      case Follower:
+        return TalonControlMode.Follower;
+      case Power:
+        return TalonControlMode.PercentVbus;
+      case Position:
+        return TalonControlMode.Position;
+      case Speed:
+        return TalonControlMode.Speed;
+      default:
+        throw new InvalidParameterException();
```
Comment (MiloDavis) [inline src/com/nutrons/framework/controllers/CanControlMode.java]: Give actual error message here

File: src/com/nutrons/framework/controllers/CanControllerProxy.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.nutrons.framework.controllers;
+
+/**
```
Comment (MiloDavis) [inline src/com/nutrons/framework/controllers/CanControllerProxy.java]: Remove created by

File: src/com/nutrons/framework/controllers/RunAtPowerEvent.java [hunk 4]
```diff
@@ -18,10 +19,13 @@ public double power() {
 
   @Override
   public void actOn(Talon talon) {
-    if(Math.abs(power) > 1.0) {
-        throw new EventUnimplementedException("Power greater than magnitude of 1.0 is not supported for Talons");
+    talon.changeControlMode(CanControlMode.Power);
+
+    if (Math.abs(power) > 1.0) {
+      throw new EventUnimplementedException(
```
Comment (MiloDavis) [inline src/com/nutrons/framework/controllers/RunAtPowerEvent.java:25]: We need an `illegalArgumentException` or an `illegalStateException` here.  This isn't quite the right thing to throw.

File: test/com/nutrons/framework/test/TestMotorEvents.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+import com.nutrons.framework.controllers.ControllerEvent;
+import com.nutrons.framework.controllers.ControllerMode;
+import com.nutrons.framework.controllers.FollowEvent;
+import com.nutrons.framework.controllers.LoopModeEvent;
+import com.nutrons.framework.controllers.RunAtPowerEvent;
+import com.nutrons.framework.controllers.Talon;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
```
Comment (MiloDavis) [inline test/com/nutrons/framework/test/TestMotorEvents.java]: Remove created by

**Review by MiloDavis — CHANGES_REQUESTED** [review]
Seems okay.  Fix things that are not

File: src/com/nutrons/framework/controllers/RunAtPowerEvent.java (hunk 4 above)
Comment (kreeve) [reply src/com/nutrons/framework/controllers/RunAtPowerEvent.java:25]: This one wasn't me, just shows up here from style changes.

**Review by MiloDavis — APPROVED** [review]

**Pull request merged**